compendium/DeclarativeServices/test/gtest/TestFixture.hpp

```diff
@@ -253,6 +253,9 @@ namespace test
         void
         TearDown() override
         {
+            if (dsRuntimeService) {
+                dsRuntimeService.reset();
+            }
```

Comment:
Is is necessary to call `reset()`? destroying the `tGenericDSSuite` object does the same thing.